js/src/widget_ngl.js

```diff
@@ -647,6 +647,49 @@ var NGLView = widgets.DOMWidgetView.extend({
         shapeComp.addRepresentation("buffer");
     },
 
+    addBuffer: function(name, kwargs){
+        console.log(name, kwargs, '0');
+        var class_dict = {
```

Comment:
@arose  can you have a look to see which class is not exposed?

Reply:
these are exposed

import ArrowBuffer from './buffer/arrow-buffer'
import BoxBuffer from './buffer/box-buffer'
import ConeBuffer from './buffer/cone-buffer'
import CylinderBuffer from './buffer/cylinder-buffer'
import EllipsoidBuffer from './buffer/ellipsoid-buffer'
import OctahedronBuffer from './buffer/octahedron-buffer'
import SphereBuffer from './buffer/sphere-buffer'
import TetrahedronBuffer from './buffer/tetrahedron-buffer'
import TextBuffer from './buffer/text-buffer'
import TorusBuffer from './buffer/torus-buffer'

Reply:
thanks